lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -53,6 +53,18 @@ const PDP_BEAM_TYPES_IN_STATISTICS_SUBQUERY = `
         LIMIT 1
     )`;
 
+/**
+ * Subquery to calculate number of runs associated with given LHC Period
+ */
+const RUNS_COUNT_SUBQUERY = `
+    (SELECT COUNT(r.run_number)
+        FROM runs as r
+        WHERE r.lhc_period_id = lhcPeriod.id
+            AND r.definition = 'PHYSICS'
+            AND r.run_quality = 'good'
+)
+`;
+
```

Comment:
Isn't there a way to do this without subquery?

Reply:
without breaking pagination or filtering or sth else, no. The same as for other subqueries